src/main/java/jenkins/plugins/office365connector/HttpWorker.java

```diff
@@ -29,89 +29,85 @@
 import org.apache.commons.httpclient.methods.StringRequestEntity;
 
 /**
- * 
  * Makes http post requests in a separate thread.
- *
  */
 public class HttpWorker implements Runnable {
 
-	private PrintStream logger;
+    private final PrintStream logger;
 
-	private String url;
+    private final String url;
+    private final String data;
+    private final int timeout;
+    private final int retries;
 
-	private String data;
+    public HttpWorker(String url, String data, int timeout, int retries, PrintStream logger) {
```

Comment:
Moved constructor to the top as it should be above rest of the method (according to at least Java convention)